### PR TITLE
Added props markerOffsetX and markerOffsetY.

### PR DESCRIPTION
**Pull request by janneklouman**
This solves #26

### CODE REVIEW

File: MultiSlider.js [hunk 1]
```diff
@@ -61,6 +64,8 @@ export default class MultiSlider extends React.Component {
       slipDisplacement: 200,
     },
     customMarker: DefaultMarker,
+    markerOffsetX: 0,
```
Comment (ptomasroos) [inline MultiSlider.js]: wouldn't it be better if you sat this to default being 24 instead and let the props being overriden when passed instead? Then you would avoid the substraction below? What do you think?

Comment (ptomasroos) [inline MultiSlider.js]: @janneklouman

**Comment by janneklouman** [comment]
Sorry for the late reply!

That was my spontaneous thought as well, but I think it's more user
friendly to have it this way:

Wouldn't it be weird to add a prop markerOffsetX={1} and all of a sudden
your marker is shifted 23 pt? To me it feels natural to have
markerOffsetX/Y={1} shift the marker 1 pt from its default position.

But its completely up to you, I can update the PR if you don't agree.

2017年8月1日(火) 14:11 Tomas Roos <notifications@github.com>:

**Comment by ptomasroos** [comment]
True makes sense. I will merge this on monday and test it out.

File: MultiSlider.js [hunk 2]
```diff
@@ -318,9 +323,9 @@ export default class MultiSlider extends React.Component {
       borderRadius: borderRadius || 0,
     };
 
-    const markerContainerOne = { top: -24, left: trackOneLength - 24 };
+    const markerContainerOne = { top: markerOffsetY - 24, left : trackOneLength + markerOffsetX - 24 }
```
Comment (ptomasroos) [inline MultiSlider.js]: Missing a ; at the end

**Pull request merged**